lessonbuilder/tool/src/webapp/templates/BltiPicker.html

```diff
@@ -119,6 +131,21 @@ <h2 rsf:id="mainhead" id="mainhead"></h2>
 		console.log(e.data);
 	}
 });
+
+// See if we are supposed to launch the shortcut link
+var shortcut_tag = document.getElementById('app-store-shortcut::blti-create::blti-create-link');
+console.log(shortcut_tag);
```

Comment:
please remove console.log

Reply:
Do I need to rename my branch or just update the original SAK- branch.

Reply:
No need to change your branch. just add more commits and push and they will be auto-updated here.